main.go

```diff
@@ -40,30 +44,86 @@ const (
 	gcsBrowserURLPrefix = "https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/logs"
 )
 
-func processJobResults(suites *junit.Suites) {
-	var passed, failed, skipped, pending int
+type JobMonitoring struct {
```

Comment:
```suggestion
type jobMonitoring struct {
```
no need for this to be public